### PR TITLE
Q4

### PR DESCRIPTION
**Pull request by aqattan**

### CODE REVIEW

File: Q4.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+JavaScript:
+
+var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var x=0;
+var isprint=true;
+while (x<a.length){
```
Comment (malkhafaji) [inline Q4.txt]: @aqattan Did you run this while loop? Did it give you the right results? I did not run it, but I know it did not :)

Comment (aqattan) [reply Q4.txt]: it prints the first word right then reverse the first and the second and so on print the third one right and then reverse the third one and fourth one.

Comment (malkhafaji) [reply Q4.txt]: @aqattan so, we need to print every other element in reverse. That is all. So,
john
ekim
susan
ttam
etc.

**Review by ssharba — APPROVED** [review]
good job

**Pull request merged**